src/py/flwr/simulation/app.py

```diff
@@ -87,6 +115,7 @@ def start_simulation(
     actor_type: Type[VirtualClientEngineActor] = ClientAppActor,
     actor_kwargs: Optional[Dict[str, Any]] = None,
     actor_scheduling: Union[str, NodeAffinitySchedulingStrategy] = "DEFAULT",
+    nodes_ids: Optional[List[int]] = None,
```

Comment:
Should we move this ahead of `client_ids`?

Reply:
I did that at first but then decided to put it at the end (just in case someone was relying on positional args when passing `clients_ids`)

Reply:
removed this arg.